src/g_game.c

```diff
@@ -439,9 +423,9 @@ static double CalcMousePitch(int mousey)
   if (!mouseSensitivity_vert_look)
     return 0.0;
 
-  pitch = I_AccelerateMouse(mousey) * (mouseSensitivity_vert_look + 5) * 8 / 10;
+  pitch = I_AccelerateMouse(-mousey) * (mouseSensitivity_vert_look + 5) * 8 / 10;
```

Comment:
Could you change the sign in `mouse_y_invert`?

Reply:
Right, so where should I add the negative sign? In the `CalcMousePitch` function?

Reply:
Is this correct? https://github.com/fabiangreffrath/woof/pull/1389/commits/ee794dc9a40ed2b69d3f8d9e8d6f72c2567138da

Reply:
> Is this correct? [ee794dc](https://github.com/fabiangreffrath/woof/commit/ee794dc9a40ed2b69d3f8d9e8d6f72c2567138da)

Yes, that's what I was thinking of earlier. Thank you.